src/Ryujinx.Memory/MemoryManagementUnix.cs

```diff
@@ -206,12 +210,14 @@ public static void UnmapSharedMemory(IntPtr address, ulong size)
 
         public static void MapView(IntPtr sharedMemory, ulong srcOffset, IntPtr location, ulong size)
         {
+#pragma warning disable CA2020 // Prevent behavioral change caused by built-in operators of IntPtr/UIntPtr
             mmap(location, size, MmapProts.PROT_READ | MmapProts.PROT_WRITE, MmapFlags.MAP_FIXED | MmapFlags.MAP_SHARED, (int)sharedMemory, (long)srcOffset);
```

Comment:
```suggestion
            mmap(location, size, MmapProts.PROT_READ | MmapProts.PROT_WRITE, MmapFlags.MAP_FIXED | MmapFlags.MAP_SHARED, sharedMemory.ToInt32(), (long)srcOffset);
```
And remove the pragma